build.gradle

```diff
@@ -6,12 +6,24 @@ repositories {
     jcenter()
 }
 
+// DEFAULT CRATE VERSION to test against
+project.ext.defaultCrateVersion = '0.54.4'
+
+task getVersion << {
+    project.ext.crateVersion = System.getProperty('crate.version', null)
```

Comment:
couldn't you set project.defaultCrateVersion as fallback instead of null? Then you don't need the code below =)